packages/x-components/tests/e2e/cucumber/tagging/tagging.feature

```diff
@@ -5,9 +5,7 @@ Feature: Tagging component
     And   a suggestions API
     And   a related tags API
     And   a recommendations API with a known response
-    And   a query tagging API
```

Comment:
These 3 steps you removed were not used anywhere else, so I think their logic can be removed from `mocked-responses.spec.ts` (unless they might be used in the future instead of the new `a tracking API with a known response`).